esp32-hal/examples/watchdog.rs

```diff
@@ -29,15 +23,12 @@ fn main() -> ! {
     );
     let mut timer0 = timer_group0.timer0;
     let mut wdt = timer_group0.wdt;
-    let mut rtc = Rtc::new(peripherals.RTC_CNTL);
-
-    rtc.rwdt.disable();
 
     wdt.start(2u64.secs());
     timer0.start(1u64.secs());
 
     loop {
-        wdt.feed();
+        // wdt.feed();
```

Comment:
left-over from testing? we could keep it but then we'd need to change the doc comment

Reply:
Every single time I've gone to test this example since it's been added I've forgotten to comment out that line first 😂  So I've just commented them all out by default. Can revert this if you think it makes sense to, but IMO commenting out that line is required to actually demonstrate the example so it should just be like that by default.

Reply:
mhhh - I think it shows how to feed the watchdog. commenting out shows what happens if you don't feed it 🤔

Reply:
Okay, I suppose I'm just misunderstanding the purpose of this example. Can revert I guess.